server/controllers/article.js

```diff
@@ -55,6 +55,105 @@ class Article {
       return badHttpResponse(response, 500, 'There was an internal server error', error);
     }
   }
+
+  /** Create a new comment
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} Comment Object
+   */
+  static async createComment(request, response) {
```

Comment:
We should have this in a comment class for the separation of concerns

Reply:
I see. Thank you.